### ISSUE TITLE
When keyboard focus is on review slider, and we press TAB key keyboard focus is not visible: A11y_.NET Core_RecipesAppMyRecipesNew_NewRecipe_Keyboard

### ISSUE THREAD

**Issue by kapilvaishna** [dotnet/maui#22822]
### Description

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."
### Tags
#A11yTCS;#A11yMAS;#WCAG2.4.3;#Keyboard;#E2E_.NETCore_Win32_Jun2024;#.NET Core;#FocusOrder;#DesktopApp;#Benchmark;#Win32;#FTP;#A11ySev3;

## Environment Details:
Application: .NET Core
Operating System: Window 11 version 23H2 OS build 22631.2861

## Repro Steps:

1. Install and open ".NET MAUI" app.
2. TAB to "My Recipes" and Press ENTER key.
3. TAB to "New" button and press ENTER key.
4. TAB till Review slider and press TAB key.
5. Observe that when keyboard focus is on review slider, and we press TAB key keyboard focus is not visible and on next tab visible on (-) decrease rating button.

## Actual Result:
When keyboard focus is on review slider, and we press TAB key keyboard focus is not visible and on next tab visible on (-) decrease rating button.
## Expected Result:
When keyboard focus is on review slider, and we press TAB key keyboard focus should move (-) decrease rating button.
## User Impact:
Keyboard only user will not be able to delete select category. 
## Recommendations:
Refer [Developer Resources (sharepoint.com)](https://microsoft.sharepoint.com/sites/accessibility/SitePages/Developer-Resources.aspx?xsdata=MDV8MDF8di1pbnNldHRAbWljcm9zb2Z0LmNvbXw4MDYwNWMyZGNkNDc0MWI5MzJiNTA4ZGJjZWQ3ZmNlNHw3MmY5ODhiZjg2ZjE0MWFmOTFhYjJkN2NkMDExZGI0N3wxfDB8NjM4MzMxMTk5MzcwMDk3NzY5fFVua25vd258VFdGcGJHWnNiM2Q4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazFoYVd3aUxDSlhWQ0k2TW4wPXwzMDAwfHx8&sdata=OUc1UFpDSGJ0VkNhaDFCRlZQVnlMRllDdUZkWm5scjFIeVR6SFZGcXA3UT0%3d&clickparams=eyAiWC1BcHBOYW1lIiA6ICJNaWNyb3NvZnQgT3V0bG9vayIsICJYLUFwcFZlcnNpb24iIDogIjE2LjAuMTcxMjYuMjAwMzIiLCAiT1MiIDogIldpbmRvd3MiIH0%3D).

### Version with bug

Unknown/Other

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Window 11 version 23H2 OS build 22631.2861

**Comment by ninachen03** [comment]
Verified on 17.11.0 Preview 1.1 (8.0.40), can repro it

**Comment by Yash14j** [comment]
GithubTags:#Rev:yaja;
